schema/deploy/shipping/views.sql

```diff
@@ -2288,15 +2286,31 @@ grant select
 
 create or replace view shipping.scan_enrollments_v1 as
 
+    with location_names as (
+        select
+            scale,
+            identifier,
+            case scale
+                when 'neighborhood_district' then details->>'name'
+                when 'puma' then details->>'NAMELSAD10'
+            end as geo_location_name
+        from warehouse.location
+        where scale in ('neighborhood_district', 'puma')
+        and hierarchy @> 'state => washington'
+    )
+
     select
         illness_questionnaire_date,
         scan_study_arm,
         priority_code,
         puma,
         neighborhood_district,
+        geo_location_name,
         sample is not null or never_tested is not null as kit_received
 
     from shipping.scan_encounters_v1
+    join location_names on coalesce(neighborhood_district,puma) = lower(location_names.identifier)
```

Comment:
I'm a little sad this is the join key, but ah well. I see why it's that way, and it's probably not worth larger reworking to make it not be. The other reason to fix it would be to provide PUMA names for outside of Washington, but I don't think that's in-scope for their dashboard right now?

Reply:
Which specific part makes you sad? The `lower()` is only needed for the neighborhood districts. 
I don't think this affects the PUMAs at all since those are just strings of integers.

Reply:
Sorry, should have been more specific. Nothing in our schema guarantees that `warehouse.location.identfier` is by itself unique. It's unique in combination with `scale`, as `(scale, identifier)` forms the PK. So joining on the partial key is prone to collision. That said, I realize in practice this is pretty unlikely to occur with the identifiers we use for those scales.

I didn't even notice the `lower()` before, but that strikes me as an oversight on our part earlier while populating the location catalog. The convention is that the values in `hierarchy` match those in `identifier` (if they exist at all).